spec/fixtures/composite-action.yml

```
@@ -0,0 +1,40 @@
+    - name: Install dependencies
+      working-directory: ./frontend
+      run: npm ci
+    - name: Build app
+      working-directory: ./frontend
+      run: npm run build
+      env:
+        APP_NAME: ${{ inputs.app-name }}
+        API_URL: ${{ inputs.api-url }}
+    - name: Upload app to AWS S3
+      working-directory: ./frontend
+      run: aws s3 sync build s3://my-bucket --delete
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)